packages/newspringchurchapp/src/ui/UserPrayer/UserPrayerCard.js

```diff
@@ -19,22 +22,58 @@ const HeaderView = styled(() => ({
   paddingBottom: 0,
 }))(PaddedView);
 
-// eslint-disable-next-line react/display-name
-const UserPrayerCard = memo(({ duration, text, ...otherProps }) => (
-  <Card {...otherProps}>
-    <HeaderView>
-      <H5>{moment(duration).fromNow()}</H5>
-    </HeaderView>
-    <CardContent>
-      <PrayerText>{text}</PrayerText>
-    </CardContent>
-  </Card>
-));
+const HorizontalTextLayout = styled(({ theme }) => ({
+  height: theme.helpers.verticalRhythm(0.875),
+}))(SideBySideView);
+
+class UserPrayerCard extends PureComponent {
+  handleShowActionSheet = () => {
+    this.ActionSheet.show();
+  };
+
+  render() {
+    const cancelIndex = 1;
+    const destructiveIndex = 0;
+    const { duration, text, id, deletePrayer, ...otherProps } = this.props;
+    const handleOnPress = (index) => {
```

Comment:
@cdanwards Shouldn't this be handled by the ActionSheet? I thought this functionality was built in?

Reply:
So, I looked into this and apparently the `onPress` event is called no matter which button is pressed, and that's the intended behavior of the package [the creator states that here.](https://github.com/beefe/react-native-actionsheet/issues/12) I just made it so that the `onPress` isn't called for the cancel button.